NEWS.md

```diff
@@ -1,6 +1,7 @@
 # babynames 0.3.0.9000
 
 * Add data validity/regression tests (@russellpierce, #27)
+* Update data for 2017 (@russellpierce #30)
```

Comment:
I've made that change.  

... also, 🍳 on my face; the lifetables ingest wasn't right.  I fixed the ingest.  The problem slipped by my notice because changes to 'last' are expected.  So, i've taken several additional steps.

1. I've added tests for the first/last in the middle of the lifetables and babynames.  Under normal circumstances that data should be static and we can use a change there to encourage additional scrutiny.  The presence of this test would have caught my error with lifetables earlier.

2.  I've added CSVs of a small sample of the lifetables and babynames data to the data-raw directory.  I think this serves to augment the first/last tests and allow for some additional visual inspection of the data in the middle.